docs/user/README.md

```diff
@@ -0,0 +1,10 @@
+> **NOTE:** This is a general template that can be used for a module README.md, example README.md, or any other README.md type in all module repositories in the Kyma organization. Use only those sections that suit your use case but keep the proposed section order.
```

Comment:
I can use it as a root repo README? I don't think so. 

The template is very explicit a module overview template. How can I use it in other scenarios then? You mean I can take over the syntactical things like heading usage.. At the moment the readme is so small, I think the note does not bring value..